src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentService.scala

```diff
@@ -125,14 +125,6 @@ class ConsignmentService(
     consignmentRepository.consignmentHasFiles(consignmentId)
```

Comment:
Looks like there are some unused constructor params now the method is remove, these should be removed as well:

- `fileMetadataRepository: FileMetadataRepository`
- `fileRepository: FileRepository`
- `ffidMetadataRepository: FFIDMetadataRepository`

Reply:
I've taken these out as well. The service is looking much tidier now.